src/app/app.component.ts

```diff
@@ -40,65 +38,36 @@ export class AppComponent {
     } else {
       console.log('Run in browser');
     }
+
+    this.components$ = this.componentRepository.getAll();
   }
 
   public onDrag(event, component): void {
     event.dataTransfer.setData("text", component.label);
   }
 
   public onDrop(event: DragEvent): void {
-    this.addComponentStream$.next(this.components.find(s => s.label === event.dataTransfer.getData("text")));
+    // this.addComponentStream$.next(this.components$.find(s => s.label === event.dataTransfer.getData("text")));
```

Comment:
Delete or?

Reply:
Ohm lets keep it as reference, it is not needed right now, but when we support composite components we need it :D